src/Helpers/FileOutput.php

```diff
@@ -67,7 +70,12 @@ public function write(string $path, array $attributes = []): void
      */
     public function assetPath(string $path): string
     {
-        $asset = Str::of(asset($path.$this->cachebusting));
+        // if path is not a url or it's local url, we will append the cachebusting
+        if (! Str::startsWith($path, ['http://', 'https://']) || Str::startsWith($path, url(''))) {
```

Comment:
```suggestion
        if (Str::startsWith($path, url(''))) {
```
Would this be enough @promatik ?

Reply:
ignore